lib/store/src/csf/testing-utils/index.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  isExportStory,
+  AnyFramework,
+  AnnotatedStoryFn,
+  StoryAnnotations,
+  ComponentAnnotations,
+  Args,
+  StoryContext,
+} from '@storybook/csf';
+
+import { prepareStory } from '../prepareStory';
+import { normalizeStory } from '../normalizeStory';
+import { normalizeProjectAnnotations } from '../normalizeProjectAnnotations';
+import { HooksContext } from '../../hooks';
+import { NormalizedProjectAnnotations } from '../..';
+
+if (process.env.NODE_ENV === 'test') {
+  // eslint-disable-next-line global-require
+  const { default: addons, mockChannel } = require('@storybook/addons');
+  addons.setChannel(mockChannel());
+}
+
+export type StoryFile = {
+  default: Record<any, any>;
+  __esModule?: boolean;
+  __namedExportsOrder?: string[];
+};
+
+type PartialPlayFn = (
+  context: Partial<StoryContext> & Pick<StoryContext, 'canvasElement'>
+) => Promise<void> | void;
```

Comment:
These types are duplicated in `@storybook/react` -- can we de-dupe?

Reply:
Consider it done!